VolumeClipWithModel/VolumeClipWithModel.py

```diff
@@ -348,38 +371,49 @@ def clipVolumeWithModel(self, inputVolume, clippingModel, clipOutsideSurface, fi
     modelToIjkTransform = vtk.vtkTransform()
     modelToIjkTransform.SetMatrix(ijkToModel)
     modelToIjkTransform.Inverse()
-    
+
     transformModelToIjk=vtk.vtkTransformPolyDataFilter()
     transformModelToIjk.SetTransform(modelToIjkTransform)
     transformModelToIjk.SetInputConnection(clippingModel.GetPolyDataConnection())
 
     # Use the stencil to fill the volume
-    
+
     # Convert model to stencil
     polyToStencil = vtk.vtkPolyDataToImageStencil()
     polyToStencil.SetInputConnection(transformModelToIjk.GetOutputPort())
     polyToStencil.SetOutputSpacing(inputVolume.GetImageData().GetSpacing())
     polyToStencil.SetOutputOrigin(inputVolume.GetImageData().GetOrigin())
     polyToStencil.SetOutputWholeExtent(inputVolume.GetImageData().GetExtent())
-    
+
     # Apply the stencil to the volume
     stencilToImage=vtk.vtkImageStencil()
     stencilToImage.SetInputConnection(inputVolume.GetImageDataConnection())
     stencilToImage.SetStencilConnection(polyToStencil.GetOutputPort())
-    if clipOutsideSurface:
-      stencilToImage.ReverseStencilOff()
-    else:
-      stencilToImage.ReverseStencilOn()
-    stencilToImage.SetBackgroundValue(fillValue)
-    stencilToImage.Update()
 
-    # Update the volume with the stencil operation result
+    # Create a copy of the input volume to work on
     outputImageData = vtk.vtkImageData()
-    outputImageData.DeepCopy(stencilToImage.GetOutput())
-    
+    outputImageData.DeepCopy(inputVolume.GetImageData())
     outputVolume.SetAndObserveImageData(outputImageData);
     outputVolume.SetIJKToRASMatrix(ijkToRas)
 
+    # Update volume with the stencil operation result depending on user choices
+    if ClipOutsideSurface:

```

Comment:
make capitalization consistent with other parts of the code (local variable names are not capitalized)